static/testdata/jail/tx-send/rx-send.js

```diff
@@ -0,0 +1,24 @@
+var status = {
+    message_id: '42',
+};
+
+function sendAsync(params) {
+    console.log("Recieving sendAsync: ", params);
+
+    var data = {
+        from: params.from,
+        to: params.to,
+        value: web3.toWei(params.value, "ether")
+    };
+
+    var hash
```

Comment:
What's the point in having not used `data` and `hash`? After deleting them the file becomes really small to live. Can we merge it to some other file?